### DISCUSSION TITLE
[Script request] Firefly III

### DISCUSSION THREAD

**Discussion by r-kane** [community-scripts/ProxmoxVE#385] [Request script]
### Application Name

Firefly III

### Website

https://www.firefly-iii.org/

### Description

> "Firefly III" is a (self-hosted) manager for your personal finances. It can help you keep track of your expenses and income, so you can spend less and save more. Firefly III supports the use of budgets, categories and tags. Using a bunch of external tools, you can import data. It also has many neat financial reports available.
from https://github.com/firefly-iii/firefly-iii

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by AdventureSeeker423** [comment]
I second this!!

**Comment by ellic052** [comment]
So it looks like they made it with an alpha release so  it isnt working properly. @MickLesk do you know if this can be fixed.

**Reply by MickLesk** [reply]
???

**Reply by MickLesk** [reply]
Its marked as Release. We dont Change our calls.

**Reply by PatrickS7** [reply]
Thanks for this script! I ran it and it does install '6.2.0-alpha.1' which is marked as 'Latest' in Releases but is noted as a '_Very early ALPHA release of Firefly III_', it looks like the latest stable is 'v6.1.25'

**Reply by MickLesk** [reply]
We dont Change any Logic. They have published AS Release and we handle this as Release
